doc/adoc/user_guide.adoc

```diff
@@ -62,9 +62,62 @@ To install the distribution migration system run:
 [listing]
 tux > sudo zypper in SLE15-Migration suse-migration-sle15-activation
 
+== Customize the Migration Process
+
+The migration system reads a custom config file from the system to upgrade.
+The contents of that file allows to influence the behavior of the upgrade
+process. Prior to the start of the upgrade process, create the following
+file:
+
+[listing]
+tux > ssh INSTANCE_USER@IP_OF_INSTANCE 'touch /etc/sle-migration-service.yml'
+
+The custom config file allows for the following settings:
+
+Enable Debug Mode::
+If enabled, prevents the upgrade system from rewinding the setup
+steps and rebooting due to a failed upgrade, allowing the issue to
+be debugged.
+
+[listing]
+debug: true|false
+
+Specify Migration Product::
+With the SLE15-Migration package a predefined product setup is
+already applied. The value for the target product can be overwritten
```

Comment:
should we not add "......for the target product in that setup can be overwritten..." ?

Reply:
I did not understand this part ?

Reply:
The predefined product setup has a target product with a default value.
That value can be overwritten, as you mentioned in that part of the docs, I wanted to
be more explicit in case it was that obvious for other readers.

Reply:
I think I got it